1.x-src/NodaTime.Tzdb/project.json

```diff
@@ -0,0 +1,40 @@
+{
+  "version": "1.2016.3-alpha",
+  "authors": [ "Jon Skeet" ],
+  "owners": [ "Jon Skeet" ],
+  "licenseUrl": "http://www.apache.org/licenses/LICENSE-2.0",
+  "projectUrl": "http://nodatime.org/",
+  "summary": "Package containing IANA time zone data ('TZDB').",
+  "description": "Package containing IANA time zone data ('TZDB'). This package version is updated whenever IANA release a new version of the data. This package targets Noda Time 1.x (from 1.3.1 onwards); the 2.x version of this package targets Noda Time 2.x.",
+  "tags": [ "iana", "timezone", "tzdb", "nodatime" ],
+  "requireLicenseAcceptance": false,
+  "repository": {
+    "url": "https://github.com/nodatime/nodatime.git"
+  },
+
+  "compilationOptions": {
+    "keyFile": "../../NodaTime Release.snk"
+  },
+
+  "compile": [
+    "../NodaTime.Tzdb/*.cs"
+  ],
+  "resource": [
+    "../../src/NodaTime/TimeZones/Tzdb.nzd"
+  ],
+  
+  "dependencies": {
+    "NodaTime": { "version": "1.3.1", "target": "package" }
```

Comment:
I did wonder about that. I haven't tried, but I suspect 1.1.0 doesn't support all the same PCL targets that we have in 1.3.1. Will give it a go.